### PR TITLE
Godot 4.2 support

### PR DESCRIPTION
**Pull request by MateusRodCosta**

### CODE REVIEW

**Comment by MateusRodCosta** [comment]
bot, build

File: org.godotengine.godot.BaseApp.yaml [hunk 1]
```diff
@@ -44,20 +47,20 @@ modules:
...
     sources:
       - type: archive
-        sha256: 3d43b2303a924816ea0e1b345ff04c9b3e27b53eadf0f26012fc0c29b019685f
-        url: https://downloads.sourceforge.net/project/scons/scons/4.4.0/SCons-4.4.0.tar.gz
+        sha256: 6e928fc97984e719814270f6863f2183b3b72180b0162a5ed09db68c9994100c
+        url: https://downloads.sourceforge.net/project/scons/scons/4.8.0/SCons-4.8.0.tar.gz
 
     build-commands:
-      - python3 -m pip install --prefix=/app .
+      - pip3 install --no-index --no-build-isolation --prefix=/app .
```
Comment (Calinou) [inline org.godotengine.godot.BaseApp.yaml:54]: Out of curiosity, what are these pip options useful for? Why weren't they needed previously?

Comment (MateusRodCosta) [reply org.godotengine.godot.BaseApp.yaml:54]: Dunno, I copied that from the Godot 4 Editor manifest.

**Review by Calinou — APPROVED** [review]
Thanks! Congratulations for your first merged pull request :tada:

**Comment by Calinou** [comment]
@overloadedorama Can you check later this week if you can use the `4.2` branch of this BaseApp from Flathub? (It'll take a few hours to be built and have the publish delay expire.)

I'm not sure if new branches are published automatically (I recall this wasn't the case a few years ago).

**Comment by OverloadedOrama** [comment]
I tested building Pixelorama 1.0 using the `4.2` branch (I removed my local build of the `4.2` branch first) and it's indeed working.

**Pull request merged**